addons/website_sale_slides/models/slide_channel.py

```diff
@@ -46,7 +46,6 @@ def write(self, vals):
 
     def _synchronize_product_publish(self):
         self.filtered(lambda channel: channel.is_published and not channel.product_id.is_published).sudo().product_id.write({'is_published': True})
-        self.filtered(lambda channel: not channel.is_published and channel.product_id.is_published).sudo().product_id.write({'is_published': False})
```

Comment:
Would it make sense to update line 43 as well, since no sync is done from unpublished channels anymore ?
like `if vals.get('is_published')` ?